src/guide/user/demo.md

```
@@ -0,0 +1,91 @@
+If you do not have access to real data, you can create your own synthetic dataset and use it to test your analysis.
+
+### 3. Connect the Data Store to the Node
+
+To enable an analysis to access the required data, the data store must be connected to the corresponding node. The process of connecting a data store to a node is described in detail [here](../admin/data-store-management).
+
+
+## Running Analyses
+
+Once the project has been created, the data uploaded to the data store, and the data store connected to the node,
+an analysis can be created within the project. Creating an analysis involves selecting the analysis code, the
+target nodes, and the entrypoint for the analysis, and is described in detail [here](./analysis).
```

markdownlint-cli2 (0.23.0)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)